doc/subsystems/networking/bsd-sockets.rst

```diff
@@ -0,0 +1,65 @@
+.. _bsd_sockets_api:
+
+BSD Sockets compatible API
+##########################
+
+Zephyr offers an implementation of a subset of BSD Sockets API (a part
+of POSIX standard). This allows to reuse existing programming experience
+and port existing simple networking applications to Zephyr.
+
+Following are the requirements and concepts which governed BSD Sockets
+compatible API implementation for Zephyr:
+
+* Should have minimal overhead, similar to the requirement for other
+  Zephyr subsystems.
+* Should be implemented on top of
+  :ref:`native networking API <networking_api_usage>` to keep modular
+  design.
+* Should be namespaced by default, to avoid name conflicts with well-known
+  names like ``close()`` which may be part of libc or other POSIX
+  compatibility libraries. If enabled, should also expose native POSIX
```

Comment:
Well, if exposure of native POSIX names is enabled, then native POSIX names should be exposed. Pretty natural use of anaphora to to avoid tautological repetition ;-). I don't have an immediate idea how to word it differently.